Sprint1/src/main/java/model/BookingManager.java

```diff
@@ -27,4 +29,18 @@ public Booking GetBookingById(int id) {
 		}
 		return null;
 	}
+	
+	private TheatreManager theatre = new TheatreManager();
```

Comment:
We don't need to create an instance of TheatreManager here since we have a static getInstance() function that get's the current static instance of TheatreManager. I will explain this singleton pattern in detail in our next meeting